cmd/bnscli/cmd_username.go

```diff
@@ -173,3 +173,41 @@ This functionality is intended to extend RegisterTokenMsg or ChangeTokenTargetsM
 	_, err = writeTx(output, tx)
 	return err
 }
+
+func cmdUpdateUsernameConfiguration(input io.Reader, output io.Writer, args []string) error {
+	fl := flag.NewFlagSet("", flag.ExitOnError)
+	fl.Usage = func() {
+		fmt.Fprintln(flag.CommandLine.Output(), `
+Create a transaction to change username extension configuration.
```

Comment:
Can you improve the printed explanation of the command?

Reply:
How can I improve it? I don't want to add configuration message details here, because those can change.

Reply:
If you know how to improve this message, I will gladly update it. 
I am merging this pull request regardless, so that I can proceed with other changes.